### ISSUE TITLE
Ability to auto-generate Subapplication #loaded, #failedRemove, and #removed methods with pharo compatible mode

### ISSUE THREAD

**Issue by marianopeck** [instantiations/tonel-vast#75]
Whenever we have specified "pharo compatible mode/strategy", we should allow to autogenerate those 3 methods as the old MonticelloImporter did.

Each generated subapplication should include class-side #loaded, #failedRemove, and #removed methods. These methods are identical for all generated Subapplications.

```smalltalk 
loaded
 "Send #initializeOnLoad to every class defined in this subapplication that implements it."

 defined do: [ :aClass |
  (aClass class includesSelector: #initializeOnLoad) ifTrue: [ aClass initializeOnLoad ] ]
```

```smalltalk
failedRemove

 self loaded
```

```smalltalk
removing
 "Send #unload to every class defined in this subapplication that implements it."

 defined do: [ :aClass |
  (aClass class includesSelector: #unload) ifTrue: [ aClass unload ] ]
```

The #loaded method depends on the special handling of class-side #initialize methods in the original (Pharo) Seaside classes. The Monticello Importer replaces #initialize methods with class-side #initializeOnLoad methods having identical content.

Generally the only sender of #initializeOnLoad in Grease and Seaside is the #loaded method.

**Comment by marianopeck** [comment]
Not sure why MonticelloImporter replaces class side `initialize` with `initializeOnLoad`

**Comment by eMaringolo** [comment]
This was solved in d90ac1d
